src/audio/module_adapter/module/generic.c

```diff
@@ -314,7 +338,8 @@ EXPORT_SYMBOL(mod_data_blob_handler_new);
  * Like fast_get() but the handler is automatically freed.
  */
 #if CONFIG_FAST_GET
-const void *mod_fast_get(struct processing_module *mod, const void * const dram_ptr, size_t size)
+const void *z_impl_mod_fast_get(struct processing_module *mod, const void * const dram_ptr,
```

Comment:
I did not have a chance to ask this during the previous review https://github.com/thesofproject/sof/pull/10379. Is it really necessary for every instance of the same module to keep its own read-only copy of data obtained via `fast_get`? This looks wasteful in terms of memory. This looks similar to `z_impl_mod_alloc_ext` in terms of whether it should be a syscall.